lib/Github/Api/GitData/Blobs.php

```diff
@@ -36,7 +36,9 @@ public function configure($bodyType = null)
      */
     public function show($username, $repository, $sha)
     {
-        $response = $this->get('repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/git/blobs/'.rawurlencode($sha));
+        $response = $this->get(
+            'repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/git/blobs/'.rawurlencode($sha)
+        );
```

Comment:
note that PSR-2 does not enforce a line length limit. It only recommends avoding too long lines. So the current formatting is already compliant.